extensions/ui/org.eclipse.smarthome.ui.basic/src/main/java/org/eclipse/smarthome/ui/basic/internal/render/AbstractWidgetRenderer.java

```diff
@@ -202,14 +208,13 @@ protected String processColor(Widget w, String snippet) {
     }
 
     protected String getCategory(Widget w) {
-        String category = escapeURL(itemUIRegistry.getCategory(w));
```

Comment:
You're right about this one. The idea was that category is used in a html data attribute so it doesn't need URL escaping, but now I see it doesn't hurt to always escape it.